src/Components/Field/Field.js

```diff
@@ -1,74 +1,57 @@
 import React from 'react'
 import './Field.css'
 
-const showField = (
-    field, 
-    harvest, 
-    seedTypes, 
-    plantSeeds,
-    watering
-)=>{ 
-    if(field.seedType !== null ){
 
-        if(field.currentWateringLevel < field.seedType.waterRequirements){
-            return (
+const waterOrHarvest = (field, harvest, water) => (
```

Comment:
Une petite signature au dessus de cette fonction ne serait pas de trop, pour qu'on sache avec quel type d'arguments on travaille ;)

Reply:
Bon okay, alors j'ai refait tous les coms. Je sais pas si tu vas en être satisfait, c'est assez lourd, mais l'avantage c'est que mtn quand je code j'ai accès aux propriétés de tous les objets qui sont en paramètres. Qu'en pense-tu :o ? Est-ce que c viable ou trop moche ?

Reply:
Why not, ca me semble très lourd a maintenir si le projet grossis, mais si tu te sent de le faire, je m'y oppose pas ;)